raft/raft_member.py

```diff
@@ -828,8 +828,8 @@ def getHydrostatics(self, rPRP=np.zeros(3), rho=1025, g=9.81):
                 My = M*dPhi_dThy
 
                 Fvec[2] += Fz                           # vertical buoyancy force [N]
-                Fvec[3] += Mx + Fz*rA[1]                # moment about x axis [N-m]
```

Comment:
This seems to make sense, but could you elaborate it further? This part of the code is quite old and I don't recall it very well.